### DISCUSSION TITLE
nsight-sys error: Cannot mix incompatible Qt library (5.15.3) with this library (5.15.2)

### DISCUSSION THREAD

**Discussion by luzdluna** [elementary/triage#339] [Q&A]
HI,
it seems that the system has been upgraded to Qt 5.15.3, but the CUDA  (nsight and related) tools stay on Qt 5.15.2. 
there will be an upgrade of the cuda tools?

thanks in advance

Elememtary Horus. 

>uname -a
Linux v 6.2.0-34-generic #34~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Thu Sep  7 13:12:03 UTC 2 x86_64 x86_64 x86_64 GNU/Linux

>nsight-sys 
QApplication: invalid style override 'adwaita' passed, ignoring it.
	Available styles: Windows, Fusion
QApplication: invalid style override 'adwaita' passed, ignoring it.
	Available styles: Windows, Fusion
OpenGL version: "4.6.0 NVIDIA 535.113.01"
QApplication: invalid style override 'adwaita' passed, ignoring it.
	Available styles: Windows, Fusion
Cannot mix incompatible Qt library (5.15.3) with this library (5.15.2)

![Screenshot from 2023-10-10 00 11 47](https://github.com/elementary/triage/assets/47133455/15f552d4-b6bc-49e2-823f-8ac8298e560c)

**Comment by RogerWeihrauch** [comment]
hi
Possible for you to stay on the elder lib?
Since this one is revealed with the CUDa package.
Possible to pin to this elder lib not the get the newer one installed?
Regards,
Roger
